libtelemetry/src/full/java/com/mapbox/android/telemetry/location/LocationEngineControllerImpl.java

```diff
@@ -85,7 +87,11 @@ private void removeLocationUpdates() {
   private PendingIntent getPendingIntent() {
     // Implicit intent is required here to work with registering receiver via context
     Intent intent = new Intent(LocationUpdatesBroadcastReceiver.ACTION_LOCATION_UPDATED);
-    return PendingIntent.getBroadcast(applicationContext, 0, intent, PendingIntent.FLAG_UPDATE_CURRENT);
+    int flags = PendingIntent.FLAG_UPDATE_CURRENT;
+    if (BuildCompat.isAtLeastS()) {
+      flags |= 33554432; //TODO: replace with PendingIntent.FLAG_MUTABLE after Android 12 release
```

Comment:
@tarigo Should we cut a ticket so we don't forget? BTW Android 12 is officially released 🎉